src/main/java/com/codeborne/selenide/drivercommands/CloseDriverCommand.java

```diff
@@ -14,10 +14,10 @@
   private final SelenideProxyServer selenideProxyServer;
   private final long closeBrowserTimeoutMs;
 
-  CloseDriverCommand(WebDriver webDriver, SelenideProxyServer selenideProxyServer, long closeBrowserTimeoutMs) {
+  CloseDriverCommand(WebDriver webDriver, SelenideProxyServer selenideProxyServer) {
     this.webDriver = webDriver;
     this.selenideProxyServer = selenideProxyServer;
-    this.closeBrowserTimeoutMs = closeBrowserTimeoutMs;
+    this.closeBrowserTimeoutMs = 5000;
```

Comment:
I suggest to remove this field at all.
There is no need to limit lifetime of `CloseDriverCommand`. Let it work as long as it takes.

Reply:
Ok